code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java

```diff
@@ -105,8 +105,9 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
                 // wait until volumesFrom/networksFrom containers start up
                 waitForDependenciesStart(instance);
 
+                progress.checkPoint("Waiting for deployment unit instances to allocate");
```

Comment:
We aren't waiting for them to be allocated, just to be past creating. If anything, the constant UNALLOCATED_WAIT_STATES is misnamed.

This comment applies to the method name changes as well.